app/uk/gov/hmrc/apiplatformadminapi/services/ApplicationsService.scala

```diff
@@ -22,13 +22,19 @@ import scala.concurrent.{ExecutionContext, Future}
 import uk.gov.hmrc.http.HeaderCarrier
 
 import uk.gov.hmrc.apiplatform.modules.common.domain.models.ApplicationId
+import uk.gov.hmrc.apiplatform.modules.common.services.EitherTHelper
 import uk.gov.hmrc.apiplatformadminapi.connectors.ThirdPartyOrchestratorConnector
-import uk.gov.hmrc.apiplatformadminapi.models.FetchedApplication
+import uk.gov.hmrc.apiplatformadminapi.models.ApplicationWithUsers
 
 @Singleton
-class ApplicationsService @Inject() (thirdPartyOrchestratorConnector: ThirdPartyOrchestratorConnector)(implicit val ec: ExecutionContext) {
+class ApplicationsService @Inject() (thirdPartyOrchestratorConnector: ThirdPartyOrchestratorConnector)(implicit val ec: ExecutionContext)
+    extends EitherTHelper[String] {
```

Comment:
I'd recommend not extending this - it's meant to be created in a scope (even if it's a whole class scope) because you might want differently typed EitherT's